index.html

```diff
@@ -510,9 +512,10 @@ <h2>Credentials</h2>
 describes properties of the credential itself such as: the <a>issuer</a>,
 the expiry time, a representative image, a public key to use for the purposes
 of verification, the revocation mechanism to use etc.
-This metadata may be signed by the issuer. A <a>verifiable credential</a> is a
-set of claims and meta data that are tamper-evident and that cryptographically
-prove who issued it.
+The credential identifier should not be revealed to the verifier if the holder
+wants to avoid coorelation. This metadata may be signed by the issuer.
```

Comment:
`coorelation` => `correlation`

I'm not sure if this is best said this way. It suggests only one of the two anti-correlation mechanisms and may fool the reader into thinking that they won't be correlated if they merely omit a credential identifier. Perhaps:

```
Credential identifiers can be used for correlation. A holder that wants to help
minimize correlation should consider using a selective disclosure scheme that
does not reveal the credential identifier and/or a terms of use policy requiring
the verifier not to correlate their information with any third parties.
```

It seems like this sort of statement should be in the privacy section, though.

Reply:
cc: @msporny ... thoughts on where this statement should go?